package v7;

/* compiled from: r8-map-id-eaaffe05e5b4975c35db95ddc7f057e1a9a0a254a43fe066fa0ad675f8b3973e */
/* loaded from: classes.dex */
public abstract class c8 {
    public static void a(int i10, int i11, int i12) {
        if (i10 < 0 || i11 > i12) {
            StringBuilder l4 = hg.k0.l("fromIndex: ", i10, ", toIndex: ", i11, ", size: ");
            l4.append(i12);
            throw new IndexOutOfBoundsException(l4.toString());
        }
        if (i10 > i11) {
            throw new IllegalArgumentException(a4.a.l(i10, i11, "fromIndex: ", " > toIndex: "));
        }
    }
}
